enterprise/snowpark-metrics.json

```diff
@@ -669,49 +639,49 @@
         "type": "query"
       },
       {
-        "hide": 2,
-        "name": "memory_usage",
-        "query": "${VAR_MEMORY_USAGE}",
-        "skipUrlSync": false,
-        "type": "constant",
         "current": {
-          "value": "${VAR_MEMORY_USAGE}",
-          "text": "${VAR_MEMORY_USAGE}",
-          "selected": false
+          "selected": false,
+          "text": "process.memory.usage",
+          "value": "process.memory.usage"
         },
+        "hide": 0,
+        "name": "memory_usage",
         "options": [
           {
-            "value": "${VAR_MEMORY_USAGE}",
-            "text": "${VAR_MEMORY_USAGE}",
-            "selected": false
+            "selected": true,
```

Comment:
we are updated selected value from false to true. Is this intended?

Reply:
it probably doesn't matter but currently as-is i get it working on fresh installs so might as well leave it there :)